src/ansys/fluent/core/solver/flobject.py

```diff
@@ -595,7 +595,14 @@ def get_state(self) -> StateT:
     def set_state(self, state: Optional[StateT] = None, **kwargs):
         """Set the state of the object."""
         with self.while_setting_state():
```

Comment:
I noticed that we don't have the with part in the new overriding `set_state()` method.